packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js

```diff
@@ -129,11 +129,12 @@ exports.questions = {
 			}
 		]
 	}),
-	// siteNoticeUpload: new MultiFileUploadQuestion({
-	// 	title: 'Site notice',
-	// 	question: 'Upload site notice',
-	// 	fieldName: 'site-notice-upload'
-	// }),
+	uploadSiteNotice: new MultiFileUploadQuestion({
+		title: 'Upload the site notice',
```

Comment:
As I mentioned on another PR (https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3046) it might make more sense to leave the title as it was given that the title is used in the summary list to describe the captured response.

Reply:
As mentioned in the same PR I am not used to receiving requirements changes as part of the PR process. I suggest liasing with the BA to get the ticket updated if you want a change.

Reply:
Sorry for unresolving this here, I've left a comment on (#3046) also this seems to be a misleading field on the ticket, the correct title field is not mentioned on the ticket, but visible in the prototype, as Paul mentions here the previous iteration was correct but i can see why this would be misleading from the ticket naming the content as the title.

Reply:
I take the JIRA ticket as the single source of truth. Again if there is a problem with the JIRA ticket please liase with the BA to get it update.